modules/concepts/overrides.md

```diff
@@ -144,6 +144,36 @@ class FrontController extends FrontControllerCore {
 }
 ```
 
+## Override a module
+
+Since version 1.6.0.11 of PrestaShop, a new feature is available that allows developers to override some parts of the modules.
+Despite a module having different files, the main one, additional classes, etc., you can override the main file, and ModuleFrontController (note: this is available since 1.7.2.0) based files.
+
+### Override module's main class by extending it
+
+To override a module's instance class, you have to extend it, giving the extended class the same name and adding the `Override` suffix:
+
+```php
+
+if (!defined('_PS_VERSION_')) {
+	exit;
+}
+class BlockUserInfoOverride extends BlockUserInfo
+{
+	public function hookDisplayNav($params)
+	{
+                 // This is only an example, please do not use HTML here :-)
+		return '<div class="header_user_info"><a>Test</a></div>';
+	}
+}
+```
+You must put the file in `/override/modules/blockuserinfo/blockuserinfo.php`
```

Comment:
It should be automatically done by modules no?

Reply:
Or should be properly explained

Reply:
It's more a "You can" instead of "you must", especially when you have multiple ways to do it.

Reply:
Maybe it's better "you need" 🤔? Because following the steps to override a module you need to do this manually (create file, etc), the other way is described after, override a module with another module XD

Reply:
@PierreRambaud ping 😊